### ISSUE TITLE
KRPano video glitches/freezes 

### ISSUE THREAD

**Issue by MortimerGoro** [MozillaReality/FirefoxReality#3383]
I'm creating a separate issue from #3363 just to track the glitches or broken video , not the app freeze.

**[Affected Versions]:**
- Firefox Reality version 10 RC5 (e84d3c92)

**[Affected Platforms]:**
- Oculus Quest - Android 7.1.1
- HTC Vive Focus Plus - Android 7.1.2
- Pico Neo2 - Android 8.1.0
- Pico G2 4K - Android 8.1.0

**[Prerequisites]:**
- The Firefox Reality browser is open.

**[Steps to reproduce]:**
1. Play the "Hello WebXR" card from Interactive Experience.
2. Exit Hello WebXR and navigate to krpano website.
3. Enter in VR in one of the examples available (example: [link](https://krpano.com/releases/1.20.6/viewer/krpano.html?xml=examples/demotour-apartment/tour.xml)).
4. Navigate the page while in immersive mode and observe the behavior.

**[Expected results]:**
- The VR environment can be navigated.

**[Actual results]:**
- After some VR navigations KRPano shows a broken frame. You can still exit immersive mode correctly.

- Attached are the screen recording and the logcat of the issue:
	- Screen Recording: [link](https://drive.google.com/open?id=1BbKLG1cRDqjkn3eTjaUqT1xNWA3yKrbi).
	- Logcat: [link](https://drive.google.com/open?id=127zzl4ZQHyV01ZLQFNEWY-NLIrkylqlv).

**Comment by MortimerGoro** [comment]
I created this bugzilla ticket:
https://bugzilla.mozilla.org/show_bug.cgi?id=1638414

**Comment by bluemarvin** [comment]
Still blocked on Gecko.
